docs/user-guide/configure-apiml-zosmf-workflow.md

```diff
@@ -124,11 +105,31 @@ This workflow step includes the list of Zowe variables for configuration and as
    8. Click each sub-step and customize component-specific variables. Click **Next** until you complete your customization of the variables.
    9. Click **Finish** to apply the values.
 
+### Define variables
+
+The first workflow in API ML configuration is to define variables.
```

Comment:
This does make sense. What you wanted to say by this? Or you accidentally skipped word "step"  and should be "The first workflow step in API..."?